### ISSUE TITLE
LogViewer Dashboard doesn't display UmbracoFile Sink -> restrictedToMinimumLevel if set

### ISSUE THREAD

**Issue by mistyn8** [umbraco/Umbraco-CMS#11626]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

v9.0.1

### Bug summary

the logLevel set on the LogViewer shows the global serilog min setting, this can be overridden by the UmbracoFile sink RestrictedToMimumLevel

as implemented in this PR
https://github.com/umbraco/Umbraco-CMS/pull/11035

https://github.com/umbraco/Umbraco-CMS/blob/837cef0b14e44a141a58af9191f25a7c6a884b26/src/Umbraco.Infrastructure/Logging/Viewer/SerilogLogViewerSourceBase.cs#L52
I think is the where the update is required.

### Steps to reproduce

Add Sink level restriction

```
 "Serilog": {
    "MinimumLevel": {
      "Default": "Information"
    },
    "WriteTo": [
      {
        "Name": "Async",
        "Args": {
          "configure": [
            {
              "Name": "Console"
            }
          ]
        }
      },
      {
        "Name": "UmbracoFile",
        "Args": {
          "RestrictedToMinimumLevel": "Warning"          
        }
      }
    ]
```

### Expected result / actual result

Logs are restricted to `Warning`, but the logViewer displays the LogLevel as `Information`

![image](https://user-images.githubusercontent.com/12418214/141486214-f26f0c38-cb1b-4b3c-81ce-186b9f980dad.png)

---
_This item has been added to our backlog [AB#15158](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/15158)_

**Comment by nul800sebastiaan** [comment]
Sorry @mistyn8 - I am a bit confused. Are you saying there's a problem with PR https://github.com/umbraco/Umbraco-CMS/pull/11035? Did it work before that?

**Comment by bergmania** [comment]
I read the problem as the dashboards just shows the default log level. But you can decided to log on a different level for just the umrbaco sink. 
I'm not sure what should be displayed. Optimally all values should..

> Default: Information
> Umbraco: Warning
> Console: Error
> ...

**Comment by mistyn8** [comment]
@bergmania yep exactly that.. @nul800sebastiaan sorry if that wasn't clear.
my thoughts would be that the log viewer is only concerned with the `umbracoFile Sink` so should show the logging level currently in use for that sink.
Otherwise is it not confusing when you show log level = **information** (from the `serilog default setting`) but there are no logs because the `RestrictedToMinimumLevel` for the `umbracoFile Sink` is set to **warning**?

**Comment by nikolajlauridsen** [comment]
Fixed in #11715 🎉
